test/plugins/test_ftintitle.py

```diff
@@ -175,7 +175,7 @@ def test_split_on_feat(self):
         parts = ftintitle.split_on_feat("Alice defeat Bob")
         assert parts == ("Alice defeat Bob", None)
 
-    def test_contains_feat(self):
+    def test_contains_feat_artist(self):
```

Comment:
Given my suggestion above, we should only have `test_contains_feat` here. Now that `for_artist=False` is hardcoded, I'd expect some of the assertions to fail here (ones that test `for_artist=True` delimiters), so you can remove them from here.